### DISCUSSION TITLE
Shelly Plus Plug S Support for Status LED

### DISCUSSION THREAD

**Discussion by ferrettifrancesco90** [mongoose-os-apps/shelly-homekit#1597] [Ideas]
Hi,
I flashed on my Shelly Plus Plug S the homekit firmware 2.13.3 build 20241203-192046/2.13.3-g1cb5084. The ring led always stays ON in red color, even though I disabled "state LED" from the new homekit firmware webpage.
Is it a bug that you can't turn off the led?

**Comment by markirb** [comment]
If I look at the code I would think so yes.

Plus Plug S uses Neopixel lights, that we currently do not support as Status LEDs.

**Comment by ferrettifrancesco90** [comment]
So could it be integrated into a future firmware?

**Reply by markirb** [reply]
Would you mind testing this and give me feedback 

(right click save as then upload via webif)
[shelly-homekit-ShellyPlusPlugS.zip](https://github.com/user-attachments/files/18269622/shelly-homekit-ShellyPlusPlugS.zip)

this should show green when on and red when off for now

**Reply by JoJo03071967** [reply]
Hello,
I have the same problem. It is not possible to unable the LED its allways red.

**Reply by markirb** [reply]
Hi please see issue section https://github.com/mongoose-os-apps/shelly-homekit/issues/1672 I will close this here.
